prow/config/plugins.yaml

```diff
@@ -5,6 +5,29 @@ owners:
   skip_collaborators:
   - "metal3-io"
 plugins:
+  Nordix:
+    plugins:
+    - approve
```

Comment:
This file would also need yaml formatter.

Reply:
Done